server/methods/accounts/accounts.app-test.js

```diff
@@ -222,133 +206,95 @@ describe("Account Meteor method ", function () {
+    it("should allow Admin to edit other user address", function () {
       sandbox.stub(Reaction, "hasPermission", () => true);
       sandbox.stub(Reaction, "hasAdminAccess", () => true);
-      let account = Factory.create("account");
-      spyOnMethod("setShipmentAddress", account.userId);
-      spyOnMethod("setPaymentAddress", account.userId);
-
-      sandbox.stub(Reaction, "getShopId", () => shopId);
-      Meteor.call("cart/createCart", account.userId, sessionId);
 
       // we put new faker address over current address to test all fields
       // at once, but keep current address._id
-      const address = Object.assign({}, account.profile.addressBook[0], getAddress());
-      Meteor.call("accounts/addressBookUpdate", address, account.userId);
+      const address = Object.assign({}, fakeAccount.profile.addressBook[0], getAddress());
+      Meteor.call("accounts/addressBookUpdate", address, fakeAccount.userId);
 
       // comparing two addresses to equality
-      account = Accounts.findOne(account._id);
+      const account = Accounts.findOne(fakeAccount._id);
       const newAddress = account.profile.addressBook[0];
       expect(_.isEqual(address, newAddress)).to.be.true;
-
-      return done();
     });
 
     it("should update fields to exactly the same what we need", function () {
-      let account = Factory.create("account");
-      sandbox.stub(Meteor, "userId", () => account.userId);
-      sandbox.stub(Reaction, "getShopId", () => shopId);
-      spyOnMethod("setShipmentAddress", account.userId);
-      spyOnMethod("setPaymentAddress", account.userId);
-      Meteor.call("cart/createCart", account.userId, sessionId);
-
       // we put new faker address over current address to test all fields
       // at once, but keep current address._id
-      const address = Object.assign({}, account.profile.addressBook[0], getAddress());
+      const address = Object.assign({}, fakeAccount.profile.addressBook[0], getAddress());
       Meteor.call("accounts/addressBookUpdate", address);
 
       // comparing two addresses to equality
-      account = Accounts.findOne(account._id);
+      const account = Accounts.findOne(fakeAccount._id);
       const newAddress = account.profile.addressBook[0];
       expect(_.isEqual(address, newAddress)).to.be.true;
     });
 
     it("should throw error if wrong arguments were passed", function () {
       const updateAccountSpy = sandbox.spy(Accounts, "update");
-      expect(() => Meteor.call("accounts/addressBookUpdate", 123456)).to.throw;
-      expect(() => Meteor.call("accounts/addressBookUpdate", {})).to.throw;
-      expect(() => Meteor.call("accounts/addressBookUpdate", null)).to.throw;
-      expect(() => Meteor.call("accounts/addressBookUpdate")).to.throw;
-      expect(() => Meteor.call("accounts/addressBookUpdate", "asdad", 123)).to.throw;
+      expect(() => Meteor.call("accounts/addressBookUpdate", 123456)).to.throw();
```

Comment:
@pmn4 Should we make these expectations more explicit while we are here? Doesn't this mask any sort of other error is thrown? I think we have done a better job lately of making these error messages more consistent. I realize that's how the existing code works but since we are changing this, maybe we can make it better?